File: BaekJoon/BruteForce/7568.py
N = int(input())

temp = []

for _ in range(N):
    (x, y) = (map(int, input().split()))
    temp.append((x,y))

result = ""
for p in temp:
    result += str(len(list(filter(lambda x: p[0]<x[0] and p[1] < x[1], temp)))+1)+" "

print(result.rstrip())

File: BaekJoonSwift/BaekJoonSwift/Bruth Force/7568.swift
import Foundation

let n = Int(readLine()!)!
var v = [[Int]]()
var rank = [Int]()
var i = 0
for _ in 0..<n {
    v.append(readLine()!.split(separator:  " ").map { Int($0)! })
    rank.append(0)
}

for li in v {
    let temp = v.filter {
        if $0[0] > li[0] && $0[1] > li[1]{
            return true
        } else {
            return false
        }
    }
    rank[i] = temp.count + 1
    i+=1
}

for item in rank {
    print("\(item)",terminator: " ")
}
